packages/terra-paginator/src/ControlledProgressivePaginator.jsx

```diff
@@ -34,6 +34,10 @@ const propTypes = {
    * The intl object to be injected for translations.
    */
   intl: PropTypes.shape({ formatMessage: PropTypes.func }).isRequired,
+  /**
+   * Allows to set custom page label.
```

Comment:
should we mention in the doc to let the consumers know that translations needs to be handled for this  ?

Reply:
Added a usage note here https://github.com/cerner/terra-core/pull/3470/commits/839d26620a815100d31caab980928c3c9755ea57